### PR TITLE
[AS-87] Connect creator to application flow

### PR DESCRIPTION
**Pull request by mslup**

### CODE REVIEW

File: src/pages/Projects.tsx [hunk 1]
```diff
@@ -59,10 +61,16 @@ const Projects = (): JSX.Element => {
...
+  const handleCreateProject = () => {
+    dispatch(clearCreatorState());
+    navigate(ROUTES.CREATOR);
+  };
+
   return (
     <div className="projects-page">
       <h2>Your latest projects</h2>
       {message}
+      <Button onClick={handleCreateProject}>Create new project</Button>
```
Comment (andrmdmd) [inline src/pages/Projects.tsx]: New project should be created when clicking on the `Editor` in the navigation tab

**Comment by andrmdmd** [comment]
1. GJ!
2. 'Delete' button in room creator still deletes other vertex than the one selected
3. Why is 'Project name' necessary in the creator screen, if the name does not carry over to the editor screen and we have to name it again?
4. The following smoke test has a bug:
```
1. Click 'Create new project'
2. Create a room in the creator
3. Add some furniture and save the project
4. Go back to 'Projects' screen and click 'Create new project' again
Expected behavior: new room can be created
Actual behavior: no new vertices can be added, only refreshing the page helps
```

**Review by andrmdmd — CHANGES_REQUESTED** [review]

File: src/api/projectsApi.ts [hunk 2]
```diff
@@ -79,20 +80,50 @@ export const fetchGLBUrl = async (
...
+      projectName,
+    );
+    const projectsRef = collection(db, 'users', userId, 'projects');
+    const projectRef = doc(projectsRef);
+
+    await setDoc(projectRef, project);
+
+    return [true, projectRef.id];
+  } catch (error) {
+    console.error('Error creating project:', error);
```
Comment (andrmdmd) [inline src/api/projectsApi.ts]: Shouldn't we move error handling outside these API functions? They should throw errors that should be handled on higher (component) levels, e.g. by showing a snack bar with the error to the user. Console errors at this level can be used for debugging but we should think about the end user at this stage of the project, so we should at least do the console logging at the component level and turn these into snackbar errors in a separate task

Comment (mslup) [reply src/api/projectsApi.ts]: [AS-113](https://ar-chitect.atlassian.net/browse/AS-113)

File: src/pages/Projects.tsx (hunk 1 above)
Comment (mslup) [reply src/pages/Projects.tsx]: [AS-114](https://ar-chitect.atlassian.net/browse/AS-114)

**Comment by mslup** [comment]
ty for catching the bugs and the feedback! for now I added 'creator' to navbar, because merging them to appear as one in navbar will require some bigger changes

**Review by andrmdmd — APPROVED** [review]

**Pull request merged**